__tests__/integration.test.ts

```diff
@@ -95,19 +98,18 @@ describe('Node host', () => {
     const childHost = cp.spawn(
       process.execPath,
       [path.join(__dirname, '..', 'bin', 'cli.js')],
-      { env: { NVIM_NODE_HOST_DEBUG: 1 } }
+      { env: { NVIM_NODE_HOST_DEBUG: 1 }, stdio: 'ignore' }
```

Comment:
Need to close/ignore all fds to prevent child host from waiting indefinitely. `childHost.kill` is not enough.